storage/reads/aggregate_resultset_test.go

```diff
@@ -0,0 +1,168 @@
+package reads_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/influxdata/influxdb/v2/models"
+	"github.com/influxdata/influxdb/v2/storage/reads"
+	"github.com/influxdata/influxdb/v2/storage/reads/datatypes"
+	"github.com/influxdata/influxdb/v2/tsdb/cursors"
+)
+
+func TestNewWindowAggregateResultSet_Tags(t *testing.T) {
+
+	newCursor := sliceSeriesCursor{
+		rows: newSeriesRows(
+			"clicks click=1 1",
+		)}
+
+	request := datatypes.ReadWindowAggregateRequest{}
+	resultSet, err := reads.NewWindowAggregateResultSet(context.Background(), &request, &newCursor)
+
+	if err != nil {
+		t.Errorf("Error creating WindowAggregateResultSet: %s", err)
```

Comment:
Error messages generally have lowercase strings for the beginning so this would probably be better as `t.Errorf("error ...")`.

Also, if there is an error, then the rest of this test will likely not succeed since you use the `resultSet`. In that case, you should use `t.Fatalf`.